layout/frontpage.php

```diff
@@ -36,7 +36,7 @@
     $marketingspot = $i + 1;
     $marketingboxes[$i]->index = $i;
     $marketingboxes[$i]->title = get_config('theme_wwu2019', 'marketing' . $marketingspot);
-    $marketingboxes[$i]->content = get_config('theme_wwu2019', 'marketing' . $marketingspot . 'content');
+    $marketingboxes[$i]->content = format_text(get_config('theme_wwu2019', 'marketing' . $marketingspot . 'content'));
```

Comment:
@tobiasreischmann do you recall whether this is the right one for filtering content (not titles)?

Reply:
If you're just thinking about `format_text`/`format_string`, the moodle docs say this:

> ### format_string()
> This function should be used to:
> - print short strings (non html) that need filter processing (activity titles, post subjects, glossary concepts...).
> 
> All enabled heading filters will be applied to the string.

(https://docs.moodle.org/dev/Output_functions#format_string.28.29)
So format_string() would be the wrong function for this occasion.